player/webpack.config.js

```diff
@@ -86,6 +86,8 @@ const APP_SCRIPTS = [
     './public/services/controllerservice.js',
     './public/js/ekstepRendererEvents.js',
     './public/js/iEvaluator.js',
+    './public/js/webView.js',
+    './public/js/mobileView.js',
```

Comment:
There should be localView.js file which should not include while building the preview build